DEVELOPING_OPAL/tools/src/main/scala/org/opalj/support/info/Purity.scala

```diff
@@ -77,9 +73,139 @@ import org.opalj.tac.fpcf.analyses.purity.L1PurityAnalysis
 import org.opalj.tac.fpcf.analyses.purity.L2PurityAnalysis
 import org.opalj.tac.fpcf.analyses.purity.LazyL1PurityAnalysis
 import org.opalj.tac.fpcf.analyses.purity.LazyL2PurityAnalysis
-import org.opalj.tac.fpcf.analyses.purity.SystemOutLoggingAllExceptionRater
 import org.opalj.util.PerformanceEvaluation.time
 import org.opalj.util.Seconds
+import org.rogach.scallop.ScallopConf
+
+/**
+ * `PurityConf` is a configuration class for parsing and managing command-line arguments related to purity analysis
```

Comment:
There's a merge conflict on this file, please merge the current state of develop into your branch and fix the conflict.